### DISCUSSION TITLE
Add one Option to Individual Profile

### DISCUSSION THREAD

**Discussion by mh254** [jilleb/MQB-FPA#14] [Q&A] [answered]
Are there any hints how to add one Option to the Individual Profil?

I got an original Dataset for my Leon Cupra build 02/2017. I tried a lot to add one Option, ask others for Help, but without success.

I flashed around 400 Datasets, but now i have no more ideas.

Is there anyone out there who can give me advice what i need to do from Scratch in my original Profile?

**Comment by mh254** [comment]
I already tried this steps:

- add a function to an unused "list_of_grouped_controls" byte (0x4CD-0x4EB)
- set a value for each available driving mode in "profile settings" (0x365-0x4CD) section
- added options to  "available_choices_at_individual_profile[6]" (0x653-0x68F) to allow control via individual mode
- Changed grouped_controls_controlbyte (0xA27) to 7F

But still it doesn't show the Internal Sound in Individual Profile Setting.

**Comment by jilleb** [comment]
Don't forget to add to the 8FB range.

Can you share your data? I'll have a look.

**Comment by mh254** [comment]
[FPA_CUPRA_300.zip](https://github.com/jilleb/MQB-FPA/files/5872721/FPA_CUPRA_300.zip)

Here are the original and my modded files.

Thanks for taking the time.

**Reply by cevallos98** [reply]
hello, can i use your file on a AA gateway?

**Reply by kbratos** [reply]
How did you ungrouped/assigned the soundaktor to itself? I'm looking to do the same as you but I can't find the way to do it.

I have a cupra ateca and the soundaktor and valves are linked to the engine

**Comment by mh254** [comment, marked as answer]
Strange, just came from my car - everything is now as expected with the V3 version from the above attached files.

Slider for DCC, Eco and Sport are not reset to D after igntion off/on and now the Interior Sound is also available.

Only thing i did now: Reset the MIB Unit....

**Reply by jilleb** [reply]
😄 hah, that happened to me sonetimes as well. The car can do some strange things sometimes! 

Sounds good!  👍

**Reply by jackylinuxcom** [reply]
> Strange, just came from my car - everything is now as expected with the V3 version from the above attached files.
> 
> Slider for DCC, Eco and Sport are not reset to D after igntion off/on and now the Interior Sound is also available.
> 
> Only thing i did now: Reset the MIB Unit....

@mh254  Hello, I am interesting to tune DCC, could  you share which address/value you enable the slider DCC in individual profile?
Thanks

**Reply by jamstonmartames** [reply]
This is also an aim of mine, to have a bit of fun with the 14.Wheel Dampening datasets. I have the C variant for an Arteon and the B variant for the SEAT Cupra's but not the VW GTI or Golf R mk7/7.5s on the B variant. Does anyone have these so I can begin? @mh254 @jilleb @Mr-MIBonk

**Reply by langers2k** [reply]
@jamstonmartames - whilst I'm not able to help you find the VW datasets, are you able to share the Arteon C variant dataset please?

**Comment by mh254** [comment]
Is there a was to see if the settings for the Interior Sound are working? My exhaust is loud so it is hard to hear a difference...

**Reply by jilleb** [reply]
Yes, you can go into the control module and check the live measurements. Most control modules have a value there about the Driving Profiles Status

**Reply by mh254** [reply]
Doesn't really work, i think there is still something missing.

The values on in the attached screenshot change between profiles, but the interiour sound is only active in Cupra-Profile, not in individuell.
![fpa_kanal](https://user-images.githubusercontent.com/17849030/106449685-8123cd80-6484-11eb-9306-6a6f20e200f9.jpeg)

**Reply by lito69** [reply]
I can try things
